webapp/crashstats/crashstats/templatetags/jinja_helpers.py

```diff
@@ -183,39 +169,14 @@ def generate_create_bug_url(
     elif op_sys in ("Windows Unknown", "Windows 2000"):
         op_sys = "Windows"
 
-    frames = None
-    threads = parsed_dump.get("threads")
-    if threads:
-        thread_index = crashing_thread or 0
-        frames = bugzilla_thread_frames(parsed_dump["threads"][thread_index])
-
-    comment = render_to_string(
```

Comment:
Rendering to a string using Jinja templates was a complete and utter nightmare. I ditched the template and rewrote the code to build the string using Python.

Reply:
Yeah. Jinja has some operators that let you strip whitespace in various directions, but the `bug_comment.txt` is complex enough I never got them to work exactly right. We'd end up with extra newlines in random places and then someone would complain.